src/main/webapp/assets/css/contactStyle.css

```diff
@@ -0,0 +1,102 @@
+/*
+ * Containers
+ */
+
+#col-container {
+  width: 80%;
+  margin: 0 10% 50px 10%;
```

Comment:
ordering :0

Reply:
OooO not sure why the linter didn't catch these but will change!